.github/workflows/publish-integration-config.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish GitLab Integration Config
+
+concurrency:
+  group: github-enterprise-connector-${{ github.ref }}
```

Comment:
Is the group the correct one or it doesn't make any difference?

Reply:
this is due to copy/paste I'll fix it